Makefile

```diff
@@ -61,7 +61,7 @@ reset-db: build-local
 .PHONY: api
 api: build-local
 	@echo "Spinning up the webserver..."
-	@$(START_APP)
+	@docker compose up $(IMAGE_NAME)
```

Comment:
This change from #429 broke `make api` because it added the `-d` flag, which caused `make teardown` to run immediately.